src/Storages/MergeTree/ParallelReplicasReadingCoordinator.cpp

```diff
@@ -512,6 +540,18 @@ void ParallelReplicasReadingCoordinator::setMode(CoordinationMode mode_)
     mode = mode_;
```

Comment:
Probably `setMode()` method was intended to be private? It does not create `pimpl`